content/_guides/helpers.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Advice for Helpers
+category: chanop
+---
+
+Communities are healthiest when everyone works together to maintain the space
+and shape the culture. If we leave all the emotional labor for Operators to
```

Comment:
Should "Operators" be capitalised here? It feels odd to me but i don't know English